docs/usage/contributing.rst

```diff
@@ -81,15 +81,12 @@ you can run this reformatting command:
 Releasing versions
 ==================
 
-Maud uses `versioneer<https://github.com/warner/python-versioneer>`_ to handle
-releases.
-
-To release a new version of Maud, first make a pull request updating the
-changelog with the new version number and explaining what has changed since the
-previous version.
+To release a new version of Maud, edit the field :code:`version` in the file
+:code:`setup.cfg`, then make a pull request with this change.
 
 Once the changes are merged into the :code:`origin/master` branch, add a tag
-with the new version number to your local :code:`master` branch:
+with the new version number to your local :code:`master` branch, for example
+like this:
```

Comment:
Pushing with `git push --tags` might add unwanted tags to the repo if the user is not careful. Maybe `git push origin "0.2.1"` is a better suggestion here.

Reply:
I really like that idea but I'd rather do it manually the first time just in case there is anything weird.

It's true, cmdstanpy is just python so it shouldn't require special builds.